packages/browser/src/browser/__tests__/integration.test.ts

```diff
@@ -897,26 +899,56 @@ describe('retries', () => {
     expect(ajs.queue.queue.getAttempts(fruitBasketEvent)).toEqual(2)
   })
 
-  it('does not queue up events when offline if retryQueue setting is set to false', async () => {
-    const [ajs] = await AnalyticsBrowser.load({ writeKey })
+  it('does not queue events / dispatch when offline if retryQueue setting is set to false', async () => {
+    const [ajs] = await AnalyticsBrowser.load(
+      { writeKey },
+      { retryQueue: false }
```

Comment:
Were we actually testing with retryQueue true before? Or is this making the test explicit?

Reply:
Yes, previously we were testing retryQueue with true here (which is the default). It only passed because the logic of the test itself was broken.